packages/react/src/theme/components/button.ts

```diff
@@ -0,0 +1,130 @@
+import { defineStyleConfig } from "@chakra-ui/react";
+import { colors } from "../foundations";
+
+const config = defineStyleConfig({
+  baseStyle: ({ colorScheme }) => ({
+    fontFamily: "Mulish",
+    fontStyle: "normal",
+    fontWeight: "700",
+    fontSize: "18px",
+    lineHeight: "28p",
+    borderRadius: "8px",
+    border: "0",
+    outline: "transparent solid 2px",
+    outlineOffset: "2px",
+    display: "inline-flex",
+    appearance: "none",
+    alignItems: "center",
+    justifyContent: "center",
+    cursor: "pointer",
+    gap: "4px",
+    textDecoration: "none",
+    backgroundColor: `${colorScheme}.500`,
```

Comment:
Det her bør vel være defaulten til Chakra, er det sånn at vi styler helt fra scratch? Lurer på om det hadde vært gunstig å ta utgangspunkt i Chakra sin defaultstil og overskrive den ved behov. Mistenker at det blir en del styr for bidragsytere å måtte legge inn ting som dette som allerede fungerer out-of-the-box

Reply:
Helt enig. Så bare ikke grundig over det. Men sykt mye jeg kunne fjerne faktisk. Fjerna en del nå!